spacetemplate/assets/agile.yaml

```diff
@@ -383,5 +355,16 @@ work_item_type_groups:
   - *storyID
   - *taskID
   - *defectID
+  - *impedimentID
   bucket: iteration
   icon: fa fa-repeat
+
+work_item_link_types:
+
+- id: "48b1b953-ddf3-473d-8527-e705f7c5f943"
+  name: Impedes
+  description: One work item item impedes another one
+  forward_name: impedes
+  reverse_name: is impeded by
+  topology: network
```

Comment:
@michaelkleinhenz @stevengutz I wonder if the "Impedes" link type is better modeled with a `dependency` topology. Compared to the `network` topology which allows essentially everything, the `dependency` topology only adds the restriction that no cycles are allowed.

Reply:
Ok, fixed.